account/manager.go

```diff
@@ -548,11 +608,20 @@ func (m *manager) resumeAccount(ctx context.Context, account *Account, // nolint
 					"be greater than 0", acctKey)
 			}
 
-			// If we have a label prefix, then we'll apply that now
-			// and also attach some additional meta data.
-			contextLabel := fmt.Sprintf(" poold -- "+
-				"AccountCreation(acct_key=%x)", acctKey)
-			label := makeTxnLabel(m.cfg.TxLabelPrefix, contextLabel)
+			// Attach additional meta data to transaction label.
+			isExpirySpend := false
+			txFee := btcutil.Amount(1)
+			balanceDiff := account.Value
+			contextLabel, err := actionTxLabel(
+				*account,
```

Comment:
We usually write the arguments a bit more compact (while still respecting the 80 character limit). And the closing parenthesis should be on a new line for multi-line arguments.
So I'd write this like this:
```
		contextLabel, err := actionTxLabel(
				*account, CREATE, isExpirySpend,
				txFee, balanceDiff,
		)
```